velox/docs/monthly-updates/january-2023.rst

```diff
@@ -0,0 +1,76 @@
+********************
+January 2023 Update
+********************
+
+Core Library
+============
+
+* Add generic writer which supports generic outputs in the simple function interface and thus unblocks authoring a large set of functions that operate on any type such as :func:`subscript`, :func:`array_concat`, :func:`array_intersect`.
+* Reject null aware right semi project joins and add support for null-aware left semi join project with filter.
+* Fix memory usage tracking in presence of async data cache failures.
+* Make memory usage tracker thread safe.
+
+
+Presto Functions
+================
+
+* Add :func:`array_frequency`, :func:`json_parse`, :func:`json_format`, :func:`shuffle`, :func:`sha1`, :func:`spooky_hash_v2_32`, :func:`spooky_hash_v2_64`, :func:`repeat`, :func:`week`, :func:`week_of_year` scalar functions.
+* Fix error handling in :func:`least`, :func:`greatest`, :func:`like`, :func:`array_sum`, :func:`array_position`, :func:`regexp_extract`, :func:`regexp_like`, :func:`regexp_extract_all` scalar functions.
+* Fix bugs in :func:`array_except`, :func:`array_intersect`, :func:`date_parse`, :func:`bitwise_left_shift`, :func:`bitwise_right_shift` scalar functions.
+* Fix crash in :func:`approx_distinct` when passing max standard error value of 0.26 .
+
+
+Spark Functions
+===============
+
+* Add support for DATE inputs to least and greatest functions.
+* Fix year function to match Spark's semantics.
+
+Hive Connector
+==============
+
+* Fix crashes when reading from an S3 file system.
+* Add support for MAP columns in Parquet reader.
+* Fix is-null and is-not-null filter pushdown for ARRAY and MAP columns.
+* Fix unnecessary copy in the native Parquet reader.
+
+
+Substrait
+=========
+
+* Update Substrait to 0.22.0.
+
+
+Performance and Correctness
+===========================
+
+* Fix multiple bugs in expression evaluation with TRY.
+* `applyFunctionWithPeeling` should flatten single argument functions input always.
```

Comment:
Can we rephrase this in user-friendly terms? CC: @laithsakka

Reply:
How about : 'Ensure single function arguments are always flattened.` ?